README.md

```diff
@@ -23,7 +23,7 @@ Distributing PDFs will get you sued for lots of money ( https://en.wikipedia.org
 * git clone this repository
 * run this from cabal `cabal run`
 * point your web browser to `localhost:3000`
-
+* run repository with nix-shell `nix-shell shell.nix --command "cabal run fermatslastmargin"`
```

Comment:
the placement of this is confusing. the list here is an ordered list of commands to run. ideally you would actually change the above line, and omit this one.

Reply:
Thank you for suggesting.